docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+* **Use Organizational Units or not?**
+
+  Some organizations might find it necessary to establish Organizational Units to create a separation between certain identities and MSPs created by a single CA.
+
+* **Database type.**
+
+  Some channels in a network might require all data to be modeled in a way [CouchDB](couchdb_as_state_database.html) can understand, while other networks, prioritizing speed, might decide that all peers will use LevelDB.
+
+* **Channels and private data.**
+
+  Some networks might decide that [channels](channels.html) are the best way to ensure privacy and isolation for certain transactions. Others might decide that a single channel, along with [private data collections](private-data/private-data.html), better serves their need for privacy.
+
+* **Container orchestration.**
+
+  Different users might also make different decisions about their container orchestration, creating separate containers for their peer process, logging for the peer, CouchDB, gRPC communications, and chaincode, while other users might decide to combine some of these processes.
+
+* **External chaincode.**
+
+  The new [external chaincode service](cc_service.html) will also be implemented by some users and not others.
+
+However and wherever your components are deployed, you will need a high degree of expertise in your management system of choice (such as Kubernetes) in order to efficiently operate your network. Similarly, the structure of the network must be designed to fit the business use case and any relevant laws and regulations government of the industry in which the network will be designed to function.
+
+This deployment guide will not go through every iteration and potential network configuration, but does give common guidelines and rules to consider.
+
+## Step two: Set up a cluster for your resources
+
+Generally speaking, Fabric is agnostic to the method used to deploy and manage it. It is possible, for example, to deploy and manage a peer from a laptop. For a number of reasons, this is likely to be unadvisable, but there is nothing in Fabric that prohibits it.
+
+As long as you have the ability to deploy containers, whether locally (or behind a firewall), or in a cloud, it should be possible to stand up components and connect them to each other. However, Kubernetes features a number of helpful tools that have made it a popular container management platform for deploying and managing Fabric networks. For more information about Kubernetes, check out [the Kubernetes documentation](https://kubernetes.io/docs). This topic will mostly limit its scope to the binaries and provide instructions that can be applied when using a Docker deployment or Kubernetes.
```

Comment:
This is the first and last mention of firewalls, which I suspect are highly likely to be encountered in most production deployments. It might be a good idea to talk about which components in an organisation need access to components from other organisations - just to give an idea of how much networking configuration is required.

For example:
- Applications using the Fabric SDK require access to all endorsing peers from all organisations and the ordering service(s) for all channels.
- Peers need access to the ordering service(s) for the channels that they are receiving new blocks from.
- etc etc

Reply:
I'm adding firewalls as a first class decision in Step 1.